exporters/otlp/otlpmetric/otlpmetrichttp/client.go

```diff
@@ -269,21 +285,38 @@ func (r *request) reset(ctx context.Context) {
 // retryableError represents a request failure that can be retried.
 type retryableError struct {
 	throttle int64
+	errMsg   string
 }
 
 // newResponseError returns a retryableError and will extract any explicit
-// throttle delay contained in headers.
-func newResponseError(header http.Header) error {
+// throttle delay contained in headers and if there is message in the response
+// body, it will be used as the error message. If errMsg is not empty, it will
+// be used as the error message instead of the standard "retry-able" failure.
```

Comment:
```suggestion
// throttle delay contained in headers.
```

This comment is about the behavior of `retryableError`, that's where it should be.

Reply:
addressed